packages/dialtone-vue2/components/popover/popover.vue

```diff
@@ -946,16 +946,61 @@ export default {
       }
     },
 
+    getReferenceClientRect (appendTo, error) {
+      const anchorReferenceRect = this.anchorEl.getBoundingClientRect();
+
+      if (this.appendTo !== 'root' || error) return anchorReferenceRect;
+
+      const iframe = appendTo.getElementsByTagName('iframe');
```

Comment:
If I'm understanding this correctly this goes to the document root and selects the first instance of iframe in the parent root document? what if there are multiple? This is the case in dialpad app with all the integrations iframes.

Reply:
You're right, I'm going to update this to match the exact iframe from where the popover is getting instanced.